CMakeLists.txt

```diff
@@ -551,11 +551,11 @@ if(ENABLE_LUA)
 		message(STATUS "Cannot find LuaJIT! Fallback to using usual Lua.")
 		find_package(Lua REQUIRED)
 		if(Lua_FOUND)
-			add_library(luajit::luajit UNKNOWN IMPORTED)
+			add_library(luajit::luajit INTERFACE IMPORTED)
 			set_target_properties(luajit::luajit PROPERTIES
-				INTERFACE_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}")
-			set_target_properties(luajit::luajit PROPERTIES
-				IMPORTED_LOCATION "${LUA_LIBRARIES}")
+				INTERFACE_INCLUDE_DIRECTORIES "${LUA_INCLUDE_DIR}"
+				INTERFACE_LINK_LIBRARIES "${LUA_LIBRARIES}"
+			)
```

Comment:
this looks weird actually. Why not simply aliasing the Lua lib target as `luajit::luajit`?

Reply:
TMU "raw" cmake build does that, e.g. when coming from Linux package manager / macos homebrew / self-built. For other platforms that use conan for dependencies it's an actual imported library.